garak/probes/jailbreakv.py

```diff
@@ -0,0 +1,208 @@
+#!/usr/bin/env python3
+
```

Comment:
Individual probes should not have `#!` entries as these are not entry points to execute the project.
```suggestion
```